.github/workflows/testing-and-deployment.yml

```diff
@@ -135,7 +135,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
```

Comment:
I assume you had to do this due to one of the libs dropping support for 3.6.  I would think we would instead merge #1716 .  Without tests in the CI, it will be inevitable that pyvista will silently break on 3.6

Reply:
I think you're right. What do you guys think, pin `hypothesis` first and then drop Python 3.6 when this gets too hard to maintain, or just merge #1716?

Reply:
I think for this PR, if it is only a few libraries that are a little out of date, I would pin to existing 3.6 compatible version.  Then the CI can be fixed while we discuss #1716 .  This PR is kind of important to get in.

Reply:
Sounds good.